drivers/dw3/dw3.c

```diff
@@ -647,6 +647,9 @@ UsbdSetupHdlr (
 	                DEBUG ((DEBUG_INFO, "USB_REQ_SET_ADDRESS\n"));
 	                if (CtrlRequest->RequestType == USB_RT_TX_DIR_H_TO_D) {
 	                        Status = UsbdSetAddress ((UINT8)CtrlRequest->Value);
+							if (EFI_ERROR (Status)) {
+	              				DEBUG ((DEBUG_INFO, "UsbdSetAddress: EFI_DEVICE_ERROR\n"));
+	        				}
```

Comment:
no return ?

Reply:
Prefer to keep the previous code flow since this might just be a warning.